src/Theming/ThemedSample.js

```diff
@@ -3,23 +3,21 @@ import * as React from "react";
 import tokens from "@kiwicom/orbit-design-tokens";
 import { withTheme } from "theming";
 
-type ComponentProps = {
-  theme: typeof tokens,
-};
+import type { Props } from "./ThemedSample";
 
-const ComponentWithTheme = ({ theme }: ComponentProps) => (
+const ComponentWithTheme = ({ theme }: Props) => (
   <div
     style={{
-      color: theme.colorTextPrimary,
-      fontFamily: theme.fontFamily,
-      lineHeight: theme.lineHeightText,
+      color: theme && theme.colorTextPrimary,
```

Comment:
I wanted do simply `const ComponentWithTheme = ({ theme }: Props) => { theme && (<div style={{...` but eslint rules are too strict. If you rather want me add if(theme) and returns instead of checking each defined, let me know.